### PR TITLE
feat: backpack v2 endpoints

### PR DESCRIPTION
**Pull request by pedrotambo**

### CODE REVIEW

File: src/controllers/handlers/emotes-handler.ts [hunk 1]
```diff
@@ -1,59 +1,60 @@
...
+  OnChainEmote,
+  OnChainEmoteResponse,
+  PaginatedResponse
+} from '../../types'
+import { createFilters } from './items-commons'
 
-// TODO: change this name
-type ItemResponse = Item & {
-  definition?: EmoteDefinition
+function mapItemToItemResponse(item: OnChainEmote, definitions: EmoteDefinition | undefined): OnChainEmoteResponse {
```
Comment (pedrotambo) [inline src/controllers/handlers/emotes-handler.ts:13]: nit: use `definition` for the variable and the 
```
  ...
  rarity: item.rarity,
  definition
}
```

File: src/controllers/handlers/errorHandler.ts [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+        }
+      }
+    }
+
+    console.log(`Error handling ${ctx.url.toString()}: ${error.message}`)
+    return {
+      status: 500,
+      body: {
+        error: 'Internal Server Error',
+        message: error.message
```
Comment (pedrotambo) [inline src/controllers/handlers/errorHandler.ts:37]: Do we want to expose the error message to the client?

File: src/controllers/handlers/explorer-handler.ts [hunk 3]
```diff
@@ -0,0 +1,170 @@
...
+  ErrorResponse,
+  HandlerContextWithPath,
+  InvalidRequestError,
+  OnChainWearable,
+  PaginatedResponse,
+  ThirdPartyWearable
+} from '../../types'
+import { createFilters } from './items-commons'
+
+const VALIDA_COLLECTION_TYPES = ['base-wearable', 'on-chain', 'third-party']
```
Comment (pedrotambo) [inline src/controllers/handlers/explorer-handler.ts:18]:
```suggestion
const VALID_COLLECTION_TYPES = ['base-wearable', 'on-chain', 'third-party']
```

File: src/controllers/handlers/explorer-handler.ts [hunk 4]
```diff
@@ -0,0 +1,170 @@
...
+): Promise<PaginatedResponse<MixedWearableResponse> | ErrorResponse> {
+  const { wearableDefinitionsFetcher } = context.components
+  const { address } = context.params
+  const pagination = paginationObject(context.url)
+  const filter = createFilters(context.url)
+  const sorting = createCombinedSorting<MixedWearable>(context.url)
+  const collectionTypes = context.url.searchParams.has('collectionType')
+    ? context.url.searchParams.getAll('collectionType')
+    : VALIDA_COLLECTION_TYPES
+  const thirdPartyCollectionId = context.url.searchParams.has('thirdPartyCollectionId')
```
Comment (pedrotambo) [inline src/controllers/handlers/explorer-handler.ts:134]: typo `thirdPartyCollectionIdS`

File: src/controllers/handlers/errorHandler.ts (hunk 2 above)
Comment (marianogoldman) [reply src/controllers/handlers/errorHandler.ts]: May be not. Removed the message.

**Review by marianogoldman — APPROVED** [review]

**Pull request merged**